pandas/tests/test_frame.py

```diff
@@ -12328,6 +12328,24 @@ def test_unstack_dtypes(self):
         expected = Series({'float64' : 2, 'object' : 2})
         assert_series_equal(result, expected)
 
+        # GH7405
+        for c, d in (np.zeros(5), np.zeros(5)), \
```

Comment:
I don't like see ``check_dtypes=False``. So you need to change the expected frame to have the dtypes you expect so this doesn't happen.